docs/configuration/action.md

```diff
@@ -4,9 +4,9 @@ title: Action
 sidebar_position: 5
 ---
 
-The action configuration allows you to automate your workflow by defining commands that are executed on an incoming event. Actions, similarly as communication platforms, use [source bindings](source/index.md) to define the event sources (command triggers) and [executor bindings](../configuration/executor) to run a given command.
+The action configuration allows you to automate your workflow by defining commands that are executed on an incoming event. Similar to communication platforms, Actions use [source bindings](source/index.md) to define the event sources (command triggers) and [executor bindings](../configuration/executor) to run a given command.
```

Comment:
Actually, we don't capitalize "actions", as you can see in this and other docs. So here's a small suggestion:

```suggestion
The action configuration allows you to automate your workflow by defining commands that are executed on an incoming event. Similar to communication platforms, actions use [source bindings](source/index.md) to define the event sources (command triggers) and [executor bindings](../configuration/executor) to run a given command.
```